source/codegen/metadata/niscope/config.py

```diff
@@ -21,41 +21,49 @@
             'fields': [
                 {
                     'grpc_name': 'absolute_initial_x',
+                    'grpc_type': 'double',
```

Comment:
Note: Ran the format_mi_metadata.py script on NI-Scope again to populate these. It was the only MI driver that had custom_types and hasn't yet been imported from a hapigen export.